sse2neon.h

```diff
@@ -4591,9 +4591,17 @@ FORCE_INLINE void _mm_sfence(void)
     __sync_synchronize();
 }
 
+// Store 128-bits (composed of 4 packed single-precision (32-bit) floating-
+// point elements) from a into memory using a non-temporal memory hint.
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_stream_ps
+void _mm_stream_ps(float *p, __m128 a)
+{
+    vst1q_f32(p, vreinterpretq_f32_m128(a));
```

Comment:
Shall we add the code like
```c
#if __has_builtin(__builtin_nontemporal_store)
    __builtin_nontemporal_store(a, (__m128 *) p);
#else 
    vst1q_f32(p, vreinterpretq_f32_m128(a));
#endif
```

Reply:
You are right. LLVM specifies `__builtin_nontemporal_store` in intrinsics header: https://github.com/llvm/llvm-project/blob/master/clang/lib/Headers/xmmintrin.h#L2138
```c
static __inline__ void __DEFAULT_FN_ATTRS
_mm_stream_ps(float *__p, __m128 __a)
{
  __builtin_nontemporal_store((__v4sf)__a, (__v4sf*)__p);
}
```